lib/ansible/modules/cloud/azure/azure_rm_virtualmachine.py

```diff
@@ -1230,17 +1233,17 @@ def exec_module(self, **kwargs):
                         vm_resource.tags = vm_dict['tags']
 
                     # Add custom_data, if provided
-                    if vm_dict['properties']['osProfile'].get('customData'):
+                    if vm_dict['properties'].get('osProfile', {}).get('customData'):
```

Comment:
pls, undo this change. as far as I remember, for VMs created from existing os disks it may cause crash